### ISSUE TITLE
external function gets called twice

### ISSUE THREAD

**Issue by Nghi93** [rubysolo/dentaku#230]
Given this hash:
```
hash = {
  test: 'POW(3, 3)'
}
calculator = Dentaku::Calculator.new  
calculator.solve(hash)  
```

my external function gets called twice with the same parameters.
Is this intended? Why is it called twice.

This does not happen when 3, 3 is given through variables:
```
calculator.store(a: 3, b: 3) 
calculator.solve(test: 'POW(a,b)')  
```

**Comment by rubysolo** [comment]
No, this is not intended -- I'll look into it.  Thanks for reporting!

**Comment by rubysolo** [comment]
Fixed in f508515

**Comment by Nghi93** [comment]
Thank you for fixing this issue so fast.
Is it possible to release a patch version for dentaku?

**Comment by rubysolo** [comment]
Sure, I'll try to get a release out this afternoon.

**Comment by rubysolo** [comment]
Released v3.4.1! 🎉
